pkg/reconciler/workload/synctargetexports/synctargetcompatible_reconcile_test.go

```diff
@@ -84,7 +84,7 @@ func TestSyncTargetCompatibleReconcile(t *testing.T) {
 				}),
 			},
 			wantSyncedResources: []workloadv1alpha1.ResourceToSync{
-				{GroupResource: apisv1alpha1.GroupResource{Group: "apps", Resource: "deployments"}, Versions: []string{"v1"}, State: workloadv1alpha1.ResourceSchemaIncompatibleState},
+				{GroupResource: apisv1alpha1.GroupResource{Group: "apps", Resource: "deployments"}, Versions: []string{"v1"}, State: workloadv1alpha1.ResourceSchemaPendingState},
```

Comment:
It seems to me that the test case title should be changed.
And probably another test case should be added for the real "incompatible" case